dv_signature_offload_api_example/sig_example.c

```diff
@@ -631,7 +652,7 @@ void dump_pi_t10dif(void *pi_ptr)
 		ntohl(pi->ref_tag));
 }
 
-void set_sig_domain_t10dif(struct mlx5dv_sig_block_domain *domain, void *sig)
+void set_sig_domain_t10dif_type1_2(struct mlx5dv_sig_block_domain *domain, void *sig)
```

Comment:
Please remove MLX5DV_SIG_T10DIF_FLAG_REF_ESCAPE from the flags because it is not relevant for type 1 and type 2. We have the same bug in the ibverbs-tests.